src/FlowAD/Types/type_utils.jl

```diff
@@ -40,8 +40,8 @@ function setup_well_model(M::jutulModel{D, T}, f::Union{jutulForce{D, T}, jutulV
     model, parameters = setup_reservoir_model(domain, sys, wells = Is)
     select_output_variables!(model.models.Reservoir, :all)
     ρ = ConstantCompressibilityDensities(p_ref = 150*bar, density_ref = [ρCO2, ρH2O], compressibility = [1e-4/bar, 1e-6/bar])
+    parameters[:Reservoir][:PhaseViscosities] = vcat(visCO2 * ones(prod(M.n))', visH2O * ones(prod(M.n))')
     replace_variables!(model, PhaseMassDensities = ρ)
-    replace_variables!(model, PhaseViscosities = vcat(visCO2 * ones(prod(M.n))', visH2O * ones(prod(M.n))'))
```

Comment:
We can keep this line still, right?

Reply:
No, this is the bad line. My understanding is that the model holds discretization-independent specification of the variables, whereas the  concrete specifications of spatial fields should go in the parameters object.